.gitignore

```diff
@@ -1,4 +1,5 @@
 *~
+*pyc
 __pycache__
```

Comment:
If you're getting `*.pyc` files tracked by git, in spite of `__pycache__` being ignored, then you are probably using python2... is that correct?